.github/workflows/release_windows_packages.yml

```diff
@@ -71,143 +71,47 @@ jobs:
         id: configure
         env:
-          PYTORCH_DEV_DOCKER: "false"
           AMDGPU_FAMILIES: ${{ inputs.families }}
+          PYTORCH_DEV_DOCKER: "false"
+          THEROCK_PACKAGE_PLATFORM: "windows"
         run: python ./build_tools/github_actions/fetch_package_targets.py
 
-  portable_linux_packages:
-    name: ${{ matrix.target_bundle.amdgpu_family }}::Build Portable Linux
-    runs-on: ${{ github.repository_owner == 'ROCm' && 'azure-linux-scale-rocm' || 'ubuntu-24.04' }}
+  windows_packages:
+    name: ${{ matrix.target_bundle.amdgpu_family }}::Build Windows
+    runs-on: ${{ github.repository_owner == 'ROCm' && 'azure-windows-scale-rocm' || 'windows-2022' }}
+    needs: [setup_metadata]
     permissions:
       contents: write
       actions: write # Added permission to trigger workflows
       id-token: write # Added permission for AWS S3 upload
-    needs: [setup_metadata]
+    defaults:
+      run:
+        shell: bash
+    strategy:
+      fail-fast: false
+      matrix:
+        target_bundle: ${{ fromJSON(needs.setup_metadata.outputs.package_targets) }}
     env:
       TEATIME_LABEL_GH_GROUP: 1
+      # TODO(scotttodd): ${OUTPUT_DIR}/${FILE_NAME} for DIST_ARCHIVE
       OUTPUT_DIR: ${{ github.workspace }}/output
-      BUILD_IMAGE: ghcr.io/rocm/therock_build_manylinux_x86_64:main
-      DIST_ARCHIVE: "${{ github.workspace }}/output/therock-dist-linux-${{ matrix.target_bundle.amdgpu_family }}${{ inputs.package_suffix }}-${{ needs.setup_metadata.outputs.version }}.tar.gz"
-      FILE_NAME: "therock-dist-linux-${{ matrix.target_bundle.amdgpu_family }}${{ inputs.package_suffix }}-${{ needs.setup_metadata.outputs.version }}.tar.gz"
+      DIST_ARCHIVE: "${{ github.workspace }}/output/therock-dist-windows-${{ matrix.target_bundle.amdgpu_family }}${{ inputs.package_suffix }}-${{ needs.setup_metadata.outputs.version }}.tar.gz"
+      FILE_NAME: "therock-dist-windows-${{ matrix.target_bundle.amdgpu_family }}${{ inputs.package_suffix }}-${{ needs.setup_metadata.outputs.version }}.tar.gz"
       RELEASE_TYPE: "${{ needs.setup_metadata.outputs.release_type }}"
       S3_TARBALL: "therock-${{ needs.setup_metadata.outputs.release_type }}-tarball"
-      S3_PYTHON: "therock-${{ needs.setup_metadata.outputs.release_type }}-python"
-    strategy:
-      fail-fast: false
-      matrix:
-        target_bundle: ${{ fromJSON(needs.setup_metadata.outputs.package_targets) }}
 
     steps:
       - name: "Checking out repository"
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
       - uses: actions/setup-python@42375524e23c412d93fb67b49958b491fce71c38 # v5.4.0
         with:
-          python-version: '3.10'
-      # TODO: We shouldn't be using a cache on actual release branches, but it
-      # really helps for iteration time.
-      - name: Enable cache
-        uses: actions/cache/restore@d4323d4df104b026a6aa633fdb11d772146be0bf # v4.2.2
-        with:
-          path: ${{ env.OUTPUT_DIR }}/caches
```

Comment:
This cache dir on Linux is specific to the scripts under Docker. On Windows we can maybe use the same ccache directory as in the CI build workflow.